### PR TITLE
MessageBox: allow showInput when message is empty

### PR DESCRIPTION
**Pull request by fisker**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md) | [Español](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.es.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer relative issues for you PR.

MessageBox: should allow showInput when message is empty

### CODE REVIEW

File: packages/message-box/src/main.vue [hunk 1]
```diff
@@ -25,9 +25,9 @@
             <i class="el-message-box__close el-icon-close"></i>
           </button>
         </div>
-        <div class="el-message-box__content" v-if="message !== ''">
+        <div class="el-message-box__content">
           <div class="el-message-box__status" :class="[ typeClass ]" v-if="typeClass && !center"></div>
```
Comment (Leopoldthecoder) [inline packages/message-box/src/main.vue]: Please also hide this `<div>` when `message` is empty.

Comment (fisker) [reply packages/message-box/src/main.vue]: add v-show and v-if ?

Comment (Leopoldthecoder) [reply packages/message-box/src/main.vue]: `v-if="typeClass && !center && message !== ''"`

Comment (fisker) [reply packages/message-box/src/main.vue]: do you mean this

```
<div class="el-message-box__content" v-if="typeClass && !center && message !== ''">
``` 

how can i hide this div ? what am I trying to do is show `el-message-box__input` even `options.message` is empty when `options.showInput` is true.

maybe this code below works. 

```
<div class="el-message-box__content" v-if="(typeClass && !center) || message !== ''" || showInput>
```

but I think it's unnecessary?

tell me if i was wrong

Comment (Leopoldthecoder) [reply packages/message-box/src/main.vue]: Change line29 from
```html
<div class="el-message-box__status" :class="[ typeClass ]" v-if="typeClass && !center"></div>
```
to
```html
<div class="el-message-box__status" :class="[ typeClass ]" v-if="typeClass && !center && message !== ''"></div>
```

My comment is right under line29.

Comment (fisker) [reply packages/message-box/src/main.vue]: sorry, did't notice that.

**Pull request merged**